bool request
---
float32 loop_radius